metaci/release/utils.py

```diff
@@ -13,6 +13,29 @@
 logger = logging.getLogger(__name__)
 
 
+def implementation_payload(role, config_item):
+    if role and config_item:
+        return {
+            "description": role,
+            "start_time": role,
+            "end_time": role,
+            "configuration_item": config_item,
+            "implementation_steps": role,
+        }
+    raise Exception("Please check your plan's role and org's configuration item.")
+
+
+def jwt_session():
```

Comment:
Maybe call this jwt_for_webhook() -- unlike the jwt_session function in CumulusCI that's used to get a JWT for authenticating to an org, this one does not get an access token back to start a session. And using a different name will help make it clearer that it's a different JWT.